glean_parser/schemas/metrics.2-0-0.schema.yaml

```diff
@@ -230,9 +230,11 @@ definitions:
         description: |
           **Required.**
 
-          A list of bug URLs (e.g. Bugzilla and Github) that are relevant to
-          this metric, e.g., tracking its original implementation or later
-          changes to it.
+          A list of bug URLs that are relevant to this metric, e.g., tracking
+          its original implementation or later changes to it.
+
+          This URL should be openly visible, and not behind authorization. (e.g.
+          a public Github repository or Bugzilla, not a private JIRA instance).
```

Comment:
We should also document this in the glean documentation e.g. https://mozilla.github.io/glean/book/reference/yaml/metrics.html#bugs